packages/ra-ui-materialui/src/input/ReferenceInput.tsx

```diff
@@ -172,28 +188,29 @@ const sanitizeRestProps = ({
     translateChoice,
     validation,
     ...rest
-}) => rest;
+}: any) => rest;
 
 export const ReferenceInputView = ({
     allowEmpty,
     basePath,
     children,
     choices,
-    classes,
-    className,
-    error,
+    classes = undefined,
```

Comment:
I don't understand why you initialize the value to `undefined`. That's the default, right?

Reply:
TypeScript. I could introduce yet another props interface though